### PR TITLE
First contributors random song

### PR DESCRIPTION
**Pull request by Parikshit-Hooda**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/falso/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #132 

## What is the new behavior?
Added files for generating random singer.

## Does this PR introduce a breaking change?
As far as I know, it doesn't introduce a breaking change. Please comment.
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: packages/falso/src/tests/song.spec.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  });
+
+  it('has no leading and trailing spaces', () => {
+    const song = randSong();
+
+    expect(song[0]).not.toBe(' ');
+    expect(song[song.length - 1]).not.toBe(' ');
+  });
+
+  // it('should create', () => {
```
Comment (NetanelBasal) [inline packages/falso/src/tests/song.spec.ts:18]: Why in comment?

Comment (Parikshit-Hooda) [reply packages/falso/src/tests/song.spec.ts:18]: Hey, that can be removed. It was in the template and I just kept it there.

File: package.json [hunk 2]
```diff
@@ -20,6 +20,8 @@
     "docs"
   ],
   "dependencies": {
+    "fs": "0.0.1-security",
```
Comment (NetanelBasal) [inline package.json:23]: revert please

File: packages/falso/src/lib/song.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+export function randSong<Options extends FakeOptions = never>(
+  options?: Options
+) {
+  const songName = () => {
+    let song = rand(data);
+
+    return song;
+  };
+
+  return fake(songName, options);
```
Comment (NetanelBasal) [inline packages/falso/src/lib/song.ts:28]: return fake(() => rand(data), options);

File: packages/falso/src/tests/song.spec.ts (hunk 1 above)
Comment (NetanelBasal) [inline packages/falso/src/tests/song.spec.ts:18]: remove

**Comment by Parikshit-Hooda** [comment]
I'll resolve in a couple of days positively.

**Pull request closed without merging**